gulpfile.js

```diff
@@ -31,7 +31,7 @@ const execa = require('execa');
 
 var prebid = require('./package.json');
 var dateString = 'Updated : ' + (new Date()).toISOString().substring(0, 10);
-var banner = '/* <%= prebid.name %> v<%= prebid.version %>\n' + dateString + '\nModules: <%= modules %> */\n';
+var banner = '/* <%= prebid.name %> v<%= prebid.version %>\n' + dateString + '*/\n';
```

Comment:
is there a reason to drop this line? having the list of modules at the top of the prebid code is really useful when debugging code and looking at it as a human

Reply:
Based on the issue, we did not want to add more to the JS than necessary by having duplicate lists. So it was removed from the comment to live solely in the array.

Reply:
My read on the issue was to drop the string so that "machines" could use an array more intelligently than hacking some text; but the text is still important for humans. The way it's implemented now makes it harder for humans to find the list, although still better than prior to the banner being there.

@bretg do you have strong feelings on this? My vote would be to leave the banner AND add the array that was added here.

Reply:
While I appreciate that the array is somewhat harder for a person to read, Prebid.js is constantly under fire for being larger than desired. An average publisher likely uses 5 bid adapters and 3-5 other modules, so the list of module names is likely between 150-200 bytes. Replicating that list seems to drive us away from the goal of making the build package smaller.

Reply:
While I don't personally think the few extra bytes is a big deal, I can appreciate that some pubs do. Can we at least add a util function that prints the list modules so I can just hit that in console rather than digging for the array in the source code?

Reply:
@smenzer I put in the other comment you can add pbjs.installedModules to the console to get the array printed.

Reply:
Ah you're right sorry I missed that.